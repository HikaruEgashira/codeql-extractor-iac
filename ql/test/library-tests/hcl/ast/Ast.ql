private import codeql.hcl.AST

query predicate hclAstNodes(HclAstNode n) { any() }
