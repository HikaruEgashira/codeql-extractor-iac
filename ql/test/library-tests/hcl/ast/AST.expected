| sample.hcl:1:1:1:9 | Identifier |
| sample.hcl:1:1:10:1 | terraform |
| sample.hcl:1:1:19:1 | ??? |
| sample.hcl:1:1:19:1 | ??? |
| sample.hcl:2:3:2:20 | Identifier |
| sample.hcl:2:3:7:3 | required_providers |
| sample.hcl:2:3:9:31 | ??? |
| sample.hcl:3:5:3:11 | Identifier |
| sample.hcl:3:5:6:5 | ??? |
| sample.hcl:3:5:6:5 | ??? |
| sample.hcl:3:15:6:5 | Object |
| sample.hcl:4:7:4:12 | Expr |
| sample.hcl:4:7:4:12 | Identifier |
| sample.hcl:4:7:4:35 | ??? |
| sample.hcl:4:17:4:35 | hashicorp/azurerm |
| sample.hcl:4:18:4:34 | Expr |
| sample.hcl:5:7:5:13 | Expr |
| sample.hcl:5:7:5:13 | Identifier |
| sample.hcl:5:7:5:26 | ??? |
| sample.hcl:5:17:5:26 | ~> 3.0.2 |
| sample.hcl:5:18:5:25 | Expr |
| sample.hcl:9:3:9:18 | Identifier |
| sample.hcl:9:3:9:31 | ??? |
| sample.hcl:9:22:9:31 | >= 1.1.0 |
| sample.hcl:9:23:9:30 | Expr |
| sample.hcl:12:1:12:8 | Identifier |
| sample.hcl:12:1:14:1 | provider |
| sample.hcl:12:10:12:18 | azurerm |
| sample.hcl:12:11:12:17 | Expr |
| sample.hcl:13:3:13:10 | Identifier |
| sample.hcl:13:3:13:13 | ??? |
| sample.hcl:13:3:13:13 | features |
| sample.hcl:16:1:16:8 | Identifier |
| sample.hcl:16:1:19:1 | resource |
| sample.hcl:16:10:16:33 | azurerm_resource_group |
| sample.hcl:16:11:16:32 | Expr |
| sample.hcl:16:35:16:38 | rg |
| sample.hcl:16:36:16:37 | Expr |
| sample.hcl:17:3:17:6 | Identifier |
| sample.hcl:17:3:17:32 | ??? |
| sample.hcl:17:3:18:22 | ??? |
| sample.hcl:17:14:17:32 | myTFResourceGroup |
| sample.hcl:17:15:17:31 | Expr |
| sample.hcl:18:3:18:10 | Identifier |
| sample.hcl:18:3:18:22 | ??? |
| sample.hcl:18:14:18:22 | westus2 |
| sample.hcl:18:15:18:21 | Expr |
